p2p/libp2p/discovery/continuousKadDhtDiscoverer.go

```diff
@@ -253,6 +251,22 @@ func (ckdd *ContinuousKadDhtDiscoverer) tryConnectToSeeder(
 	chanDone <- struct{}{}
 }
 
+func printConnectionErrorToSeeder(peer string, err error) {
+	if errors.Is(err, p2p.ErrUnwantedPeer) {
+		log.Trace("error connecting to seeder",
```

Comment:
the message could be `tryConnectToSeeder: unwanted peer` in order to differentiate it from the Debug one

Reply:
done